src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -1564,6 +1564,7 @@ size_t InterpolateKey::hash() const {
 
     size_t seed = 0;
 
+    seed = hash_combine(seed, nodeAttrs.shapeCalcMode);
```

Comment:
Why you added it into hash computation? I believe JIT kernel doesn't depend on it

Reply:
reverted this line

Reply:
Comparison operator should be supported as well

![image](https://github.com/user-attachments/assets/e31f4c92-56dc-42b8-b317-aea400701342)

Reply:
addressed